CMakeLists.txt

```diff
@@ -230,11 +223,11 @@ ELSEIF(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -static -static-libgcc -static-libstdc++")
     set(CMAKE_STATIC_LIBRARY_FLAGS "${CMAKE_STATIC_LIBRARY_FLAGS} -static -static-libgcc -static-libstdc++")
 
-    if (NOT CMAKE_TOOLCHAIN_FILE)
-        # Only include these if we are NOT cross-compiling on Linux
-        include_directories(${CMAKE_CURRENT_SOURCE_DIR}/external/mingw-w64-headers)
-        link_directories(${CMAKE_CURRENT_SOURCE_DIR}/external/mingw-w64-headers)
-    endif()
+    #if (NOT CMAKE_TOOLCHAIN_FILE)
```

Comment:
Alex said it was no longer needed, and my preliminary testing suggests that's true. However, I would like to test it more before it goes upstream. Sorry about that.